packages/core/src/components/card/_overrides.scss

```diff
@@ -0,0 +1,47 @@
+.#{$ns}-card {
+  border: 1px solid $card-background-color;
+
+  &.#{$ns}-dark,
+  .#{$ns}-dark & {
+    border: 1px solid $dark-card-background-color;
+  }
+
+  &:hover {
+    border-color: $card-hover-border-color;
+
+    &.#{$ns}-dark,
+    .#{$ns}-dark & {
+      border: 1px solid $dark-card-hover-border-color;
+    }
+  }
+
+  &.#{$ns}-disabled {
+    pointer-events: none;
+
+    > * {
+      opacity: .85;
+    }
+
+    &.#{$ns}-dark,
+    .#{$ns}-dark & {
+      > * {
+        opacity: .6;
+      }
+    }
+
+    &.#{$ns}-interactive:hover {
+      box-shadow: none;
+      cursor: auto;
+    }
+  }
+
+  &.#{$ns}-kind-g-active,
+  &.#{$ns}-kind-g-active:hover {
+    border-color: $g-primary3;
```

Comment:
In the design, they specify that the active stoke color is the primary700:
![image](https://user-images.githubusercontent.com/36133677/179714133-77830910-817c-4753-8566-4487105903b4.png)

The primary colors are not added yet but if this design needs them, I think we should add them here!

Reply:
Added with [b1f8398](https://github.com/graphext/blueprint/pull/61/commits/b1f83981aab938d6ee56f34907f6c7a4a4b81d8c)